topics/admin/tutorials/ansible-galaxy/tutorial.md

```diff
@@ -732,37 +756,52 @@ For this, we will use NGINX. It is possible to configure Galaxy with Apache and
 >
 >    {% raw %}
 >    ```nginx
->    ##
->    ## This file is maintained by Ansible - CHANGES WILL BE OVERWRITTEN
->    ##
 >    server {
+>        # Listen on port 443
 >        listen        *:443 ssl default_server;
+>        # The virtualhost is our domain name
 >        server_name   "{{ inventory_hostname }}";
 >
+>        # Our log files will go here.
 >        access_log  /var/log/nginx/access.log;
 >        error_log   /var/log/nginx/error.log;
 >
+>        # The most important location block, by default all requests are sent to uWSGI
 >        location / {
+>            # This is the backend to send the requests to.
 >            uwsgi_pass 127.0.0.1:8080;
 >            uwsgi_param UWSGI_SCHEME $scheme;
 >            include uwsgi_params;
 >        }
 >
+>        # Static files can be more efficiently served by Nginx. Why send the
+>        # request to uWSGI which should be spending its time doing more useful
+>        # things like serving Galaxy!
 >        location /static {
 >            alias {{ galaxy_server_dir }}/static;
 >            expires 24h;
 >        }
 >
+>        # The style directory is in a slightly different location
 >        location /static/style {
 >            alias {{ galaxy_server_dir }}/static/style/blue;
 >            expires 24h;
 >        }
 >
+>        # Likewise for compiled JS, these need to be accesses in a specific location.
 >        location /static/scripts {
 >            alias {{ galaxy_server_dir }}/static/scripts;
 >            expires 24h;
 >        }
```

Comment:
I don't think this block is necessary, it is not part of our docs anymore: https://docs.galaxyproject.org/en/master/admin/nginx.html#serving-galaxy-at-the-web-server-root